conda-recipes/httplib2-feedstock/LICENSE

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge, IBM
```

Comment:
Do we need to update this to 2020?

Reply:
No, this comes from httplib2. I don't think we should update this.

Reply:
ok